docs/modules/mongodb.md

```diff
@@ -0,0 +1,69 @@
+# MongoDB
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for MongoDB.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the MongoDB module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/mongodb
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a MongoDB container](../../modules/mongodb/mongodb.go)
+<!--/codeinclude-->
+
+<!--codeinclude-->
+[Test for a MongoDB container](../../modules/mongodb/mongodb_test.go)
+<!--/codeinclude-->
+
+## Module reference
+
+The MongoDB module exposes one entrypoint function to create the MongoDB container, and this function receives two parameters:
+
+```golang
+func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*MongoDBContainer, error)
+```
+
+- `context.Context`, the Go context.
+- `testcontainers.ContainerCustomizer`, a variadic argument for passing options.
+
+### Container Options
+
+When starting the MongoDB container, you can pass options in a variadic way to configure it.
+
+#### Image
+
+If you need to set a different MongoDB Docker image, you can use `testcontainers.WithImage` with a valid Docker image
+for MongoDB. E.g. `testcontainers.WithImage("mongo:6")`.
+
+#### Wait Strategies
+
+If you need to set a different wait strategy for MongoDB, you can use `testcontainers.WithWaitStrategy` with a valid wait strategy
+for MongoDB.
+
+!!!info
+    The default deadline for the wait strategy is 60 seconds.
+
+At the same time, it's possible to set a wait strategy and a custom deadline with `testcontainers.WithWaitStrategyAndDeadline`.
+
+#### Docker type modifiers
+
+If you need an advanced configuration for MongoDB, you can leverage the following Docker type modifiers:
+
+- `testcontainers.WithConfigModifier`
+- `testcontainers.WithHostConfigModifier`
+- `testcontainers.WithEndpointSettingsModifier`
+
+Please read the [Create containers: Advanced Settings](../features/creating_container.md#advanced-settings) documentation for more information.
+
+### Container Methods
+
+The MongoDB container exposes the following methods:
```

Comment:
Last thing to merge this PR: could you please add here the container method added, i.e. the ConnectionString? 🙏 You could use MySQL as reference: https://golang.testcontainers.org/modules/mysql/#connectionstring

I think once we have this in place, we are good to go! 🚀

Reply:
Added. Please review the additional content to ensure its accuracy and completeness. Let me know if any further additions are required.

Reply:
I code snippet would be much appreciated 🙏